frontend/src/metabase/ui/components/feedback/Skeleton/Skeleton.styled.tsx

```diff
@@ -16,7 +16,7 @@ export const getSkeletonOverrides = (): MantineThemeOverride["components"] => {
         return {
           // We replace Mantine's pulsing animation with a shimmer animation
           root: {
-            backgroundColor: "rgba(245,248,250,1.0)",
+            backgroundColor: "var(--mb-color-background-disabled)",
```

Comment:
Allows [loading skeletons to be themed](https://www.loom.com/share/ac01f71df753402abc4592577294f8f1?sid=7c71ef7a-abf6-417c-bb0f-f9e6b08c2b96). The color of `background-disabled` is the closest possible to `rgba(245, 248, 250, 1.0)` and also makes semantic sense.